includes/definitions/ciena-saos.yaml

```diff
@@ -1,9 +0,0 @@
-os: ciena-saos
```

Comment:
What's up with this?

Reply:
I'm trying to work with 2 branches at the same time... Not sure why I'm failing. Will clean this up. Perhaps in a new PR.